plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/CallTemplateMediatorExtFactory.java

```diff
@@ -65,7 +65,7 @@ private void buildParameters(OMElement elem, InvokeMediator invoker) {
             OMElement child = (OMElement) subElements.next();
             if (child.getQName().equals(new QName(XMLConfigConstants.SYNAPSE_NAMESPACE, "with-param"))) {
                 OMAttribute paramNameAttr = child.getAttribute(ATT_NAME);
-                Value paramValue = new ValueFactory().createValue("value", child);
+                Value paramValue = new ValueExtFactory().createValue("value", child);
```

Comment:
Shall we add method comments stating which throws exception on validation and which is not

Reply:
Fixed ...